src/static/reducers/cards.js

```diff
@@ -52,6 +65,48 @@ export default function cardsReducer(state = initialState, action) {
                 isCreatingCard: false
             });
 
+        case CARD_DELETE_REQUEST:
+            return Object.assign({}, state, {
+                isDeletingCard: true
+            });
```

Comment:
return {
  ...state,
  isDeletingCard: true,
}

Reply:
Same for the others! Easier to read:
 - copy the original state
 - update the key isDeletingCard to true

Reply:
OK